data/foss2cyber.json

```
@@ -0,0 +1,63 @@
+{
+    "name": "Ishan Velle",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "Front-end Web Developer | Self-taught | Designer | Freelancer | Blogger",
+    "avatar": "https://github.com/foss2cyber.png"
+    "tags": [
+      "FOSS",
+      "Web Developement",
+      "Freelancing",
+      "GNU/Linux",
+      "Homelab",
+      "Self-hosting",
+      "Infosec",
+      "OSINT/DFIR",
+    ],
+    "socials": [{
+            "icon": "FaTwitter",
+            "url": "https://twitter.com/thefoss_m"
+        },
+        {
+            "icon": "FaGithub",
+            "url": "https://github.com/foss2cyber"
+        },
+        {
+            "icon": "SiLinkedin",
+            "url": "https://linkedin.com/in/ishan-velle-fossie8"
+        },
```

Another trailing comma here `,`
```suggestion
        }
```